packs/ansible/actions/lib/shell.py

```diff
@@ -17,9 +17,14 @@ def replace_args(attribute=None):
     def _replace_args(f):
         @functools.wraps(f)
         def _wrapper(self, *args):
+            def _replace(arg):
+                for rule in rules:
+                    if arg.startswith(rule):
```

Comment:
I changed the order so that `'--verbose=vvvv': '-vvvv'` would resolve first and not propagate to the "lesser" settings.